apps/webhooks.md

```diff
@@ -0,0 +1,90 @@
+* Access token: received during [app installation](page:apps-install)
+* SSL setup.
+
+## Ready to start creating a webhook
+
+To create a webhook, make a [`POST` request](page:apps-api-post-shopid-webhooks-information) to the respective webhook resource, including:
+
+* The app's `callbackUrl`
+* The `eventType`.
+
+By default, new webhooks will be set to **active**.
+If you want a webhook to be inactive initially, pass the following key-value `"active": false` with the request.
+
+An `HTTP 201` response indicates that the webhook was successfully set.
+
+## Webhook event types
+
+* APP_UNINSTALL
+* ORDERS_CREATE
+* ORDERS_UPDATE
+* ORDERS_DELETE
+
+## Receive webhook callbacks
+
+In order to receive the callback we send whenever events are triggered, you’ll need to create an application and configure your server.
+
+Depending on server traffic, notifications on events might take a few seconds.
+
+ePages will deactivate webhooks if some events don't go through.
+You'll actively need to activate the webhook again by passing the key-value `"active": true` with a [`POST` webhooks/{webhookId}](page:apps-api-post-shopid-webhooks-webhookid-information) request.
+
+## Respond to webhook callbacks
+
+To acknowledge that you received the webhook without issue, your server should return an HTTP 200.
+Any response code outside the 200 range indicates us that you did not receive the requested webhook.
+When a webhook is not received (for whatever reason), here's is how we retry:
+
+Of course, will do our best to deliver the events via the webhooks to your callback URL.
+
+* If a webhook times out or returns a response code outside the 200 range the application will be disabled for 60 seconds.
+* Webhooks created during that 60 seconds time frame will be queued up for the next retry attempt so that these webhooks won't be lost.
+
+The maximum retry limit is 11 retries that are set as follows:
+
+* 60 seconds after the last failure
+* 180 seconds after the last failure
+* 300 seconds after the last failure
+* 600 seconds after the last failure
+* 900 seconds after the last failure
+* 1800 seconds after the last failure
+* 3600 seconds after the last failure
+* 7200 seconds after the last failure
+* 21600 seconds after the last failure
+* 50400 seconds after the last failure
+* 86400 seconds after the last failure.
+
+After the last retry attempt, the webhook will be deactivated.
+We will send a notification to the email address that you used for the registration to our Developer Program.
+To activate the webhook again, set `active` to `true` using the [`POST` webhooks/{webhookId}](page:apps-api-post-shopid-webhooks-webhookid-information) request.
+
+## Update or delete a webhook
+
+To update or delete a webhook, using your `access_token`, use the [`POST` webhooks/{webhookId}](page:apps-api-post-shopid-webhooks-webhookid-information) or [`DELETE` webhooks/{webhookId}](page:apps-api-delete-shopid-webhooks-webhookid-information) respectively.
```

Comment:
remove " using your `access_token`," because every request requires a token, we don't need to mention this here again